Cargo.toml

```diff
@@ -115,7 +115,7 @@ name = "search_msgs"
 harness = false
 
 [features]
-default = []
+default = ["vendored"]
```

Comment:
LGTM except for this irrelevant change from https://github.com/deltachat/deltachat-core-rust/pull/2526